src/main/java/io/cdap/plugin/zuora/client/ZuoraRestClient.java

```diff
@@ -0,0 +1,380 @@
+/*
```

Comment:
Our goal is to make reviewing this as simple as possible for Google. Please as a description to PR here:
- add detailed general overview on how your code works in general. Like a high abstraction view.
- And also which classes/files are auto-generated to they can skip reviewing it 
- Also source and sink should be separate PR IMO, for simplicity of review.